src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.cpp

```diff
@@ -135,9 +135,10 @@ bool& GetIsInPayload() {
 
 // Initialize the LockFreeUserSpaceInstrumentationEventProducer and establish the connection to
 // OrbitService.
-void InitializeInstrumentation() { GetCaptureEventProducer(); }
+[[gnu::visibility("default")]] void InitializeInstrumentation() { GetCaptureEventProducer(); }
```

Comment:
I think we should add a comment in this file explaining why this is necessary, possibly short if it points to the CMake file.

Reply:
done.